src/wrap-store/wrapStore.js

```diff
@@ -109,6 +111,7 @@ export default (store, {
   /**
    * Setup extended external connection 
    */
-  chrome.runtime.onConnectExternal.addListener(connectState);
-
+  if (chrome.runtime.onConnectExternal) {
```

Comment:
I agree. The feature was the ability to have scripts running on the website itself communicate with the background store.